### PR TITLE
MAJOR CHANGE: change elem names to match HTML counterparts; change mo…

### PR DESCRIPTION
**Pull request by 4ok**
…ds to elemMods; change element name 'code' to 'blockcode' for block of code; remove output of 'javascript' type in 'code' element as is; add escaping of unescaped code in 'code' element

### CODE REVIEW

File: rules/default.js [hunk 1]
```diff
@@ -5,26 +7,23 @@ module.exports = function (options) {
     // Block level
 
     code : function (code, lang, escaped) {
-        var result;
-
-        // VERY STRANGE CODE, WHY DO WE PROCESS 'javascript' AS IS ?
-        if ('javascript' == lang) {
```
Comment (4ok) [inline rules/default.js:11]: maybe then so:
`if (lang == 'bemjson') {`
to insert any bemjson content?

I save "content" block in db, and I want to save any bemjson content too

File: rules/default.js [hunk 2]
```diff
@@ -50,10 +49,7 @@ module.exports = function (options) {
 
     heading : function (text, level, raw) {
         var result = {
-            elem : 'header',
-            mods : { // BUG: MUST BE elemMods
-                level : level
-            },
+            elem : 'h' + level,
```
Comment (4ok) [inline rules/default.js:52]: maybe add:

```
mix: {
    elem: 'header',
}
```

or

```
mix: {
    elem: 'h',
}
```

for writing common css for all headers, for example — common font-family

File: rules/default.js [hunk 3]
```diff
@@ -78,20 +74,21 @@ module.exports = function (options) {
 
     list : function (body, ordered) {
         var result = {
-            elem    : 'list',
```
Comment (4ok) [inline rules/default.js:81]: maybe add:

```
mix: {
    elem: 'list',
}
```

for writing common css for all lists, for example — common font-family or padding

File: rules/default.js [hunk 4]
```diff
@@ -174,19 +171,21 @@ module.exports = function (options) {
...
-            result.tag = 'td';
+        } else {
+            result.elem = 'td';
+
+            if (tag) {
+                result.tag = 'td';
+            }
         }
 
         return result;
```
Comment (4ok) [inline rules/default.js:191]: maybe add:

```
mix: {
    elem: 'cell',
}
```

or

```
for writing common css for all table cells, for example — common padding
```

File: rules/default.js (hunk 1 above)
Comment (vithar) [inline rules/default.js:11]: Purpose of `code` function to output code sample in markdown. If you want to insert plain `BEMJSON` to output and then process it you need another solution.

File: rules/default.js (hunk 2 above)
Comment (vithar) [inline rules/default.js:52]: I see no problem in such css code:

``` css
.content__h1,
.content__h2,
.content__h3,
.content__h4,
.content__h5
content__h6
{
    font-family: sans-serif;
}
```

File: rules/default.js (hunk 3 above)
Comment (vithar) [inline rules/default.js:81]: I see no problem in such css code:

``` css
.content__ul,
.content__ol
{
    font-family: sans-serif;
}
```

File: rules/default.js (hunk 4 above)
Comment (vithar) [inline rules/default.js:191]: I see no problem in such css code:

``` css
.content__th,
.content__td
{
    font-family: sans-serif;
}
```

File: rules/default.js (hunk 2 above)
Comment (4ok) [inline rules/default.js:52]: I like to put  all techs into elems folders, for example I put all headers css in to content__header

**Pull request merged**